image.yaml

```diff
@@ -0,0 +1,37 @@
+schema_version: 1
+
+name: "rhpam-7-tech-preview/business-automation-operator"
+description: "Red Hat Business Automation Operator"
+version: "x.x"
```

Comment:
Can't you just remove this line? Is it perhaps required?

Reply:
it's required... just verified.